.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node
+node_js:
+- 14.15.1
+addons:
+  ssh_known_hosts:
+  - 134.122.39.75
+branches:
+  only:
+  - main
+  - ci_cd_liwg
```

Comment:
Wasn't `ci_cd_liwg` temporary here?

Reply:
Yes, It was temporary, but then I gave a thought that if we want to update the ci_cd, we can utilize this branch in future?